pkg/reconcilermanager/controllers/rootsync_controller_test.go

```diff
@@ -70,6 +72,15 @@ const (
 
 var rootReconcilerName = core.RootReconcilerName(rootsyncName)
 
+func role(t *testing.T, name string, opts ...core.MetaMutator) *rbacv1.Role {
+	t.Helper()
```

Comment:
i dont think we need the t.Helper() on functions that can't call t.Fail

Reply:
removed